docs/changelog.rst

```diff
@@ -4,6 +4,19 @@
 Changes in nbformat
 =========================
 
+4.3
+===
+
+`4.3 on GitHub <https://github.com/jupyter/nbformat/milestone/7>`__
+
+- A new pluggable ``SignatureStore`` class allows specifying different ways to
+  record the signatures of trusted notebooks. The default is still an SQLite
```

Comment:
My understanding is that "ess queue ell" is official (for SQL at least), but "sequel" is quite common as well. For completeness, I say "ess queue light", so we've got all the bases covered.